packages/eslint-plugin/src/rules/sort-interface-members.ts

```diff
@@ -0,0 +1,212 @@
+/**
+ * @fileoverview Forbids unsorted interface members
+ */
+
+import * as util from '../util';
+import {
+  Identifier,
+  TSCallSignatureDeclaration,
+  TSConstructSignatureDeclaration,
+  TSIndexSignature,
+  TSMethodSignature,
+  TSPropertySignature,
+  TSTypeAnnotation,
+  TSTypeReference,
+  TypeElement,
+} from '@typescript-eslint/typescript-estree/dist/ts-estree/ts-estree';
```

Comment:
whilst `typescript-estree` does have this file in its directory tree, you should avoid deep importing it because its exact path is an implementation detail of the package.

Instead `import { TSESTree } from '@typescript-eslint/typescript-estree';`